extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/KubernetesTaskRunner.java

```diff
@@ -338,7 +338,7 @@ public Map<String, Long> getTotalTaskSlotCount()
   public Collection<? extends TaskRunnerWorkItem> getKnownTasks()
   {
     List<TaskRunnerWorkItem> result = new ArrayList<>();
-    for (Pod existingTask : client.listPeonPods()) {
+    for (Job existingTask : client.listAllPeonJobs()) {
```

Comment:
The `listPeonPods()` and `listPeonPods(Set<PeonPhase> phases)` methods will be unused if you make this change right?  If so please remove them.

Reply:
this is done